### DISCUSSION TITLE
Running customised code in another folder

### DISCUSSION THREAD

**Discussion by Turbulence1116** [byuflowlab/FLOWUnsteady#83] [Q&A] [answered]
Hi all, 

I would like to run customised codes out of the ./julia path, say in ~/App/flow_unsteady. How should I run the code within julia terminal? I tried the following:

import FLOWUnsteady as uns

include(~/App/flow_unsteady, "wing.jl")

It did not work, / is not a unary operator. Can I have any hint pls? Thank you.

**Comment by jayjiejie** [comment]
Perhaps try to use the absolute path, e.g. /home/username/App/flow_unsteady/example/wing instead?

**Comment by Turbulence1116** [comment]
yeah I'll give it a try today and get back to you later.

**Reply by Turbulence1116** [reply]
The right way is to cd in the directory where the .jl file is located, then launch julia, import FLOWUnsteady, include("wing.jl"). This way the code can be run in any directory/folder.

**Reply by EdoAlvarezR** [reply, marked as answer]
You can also give the path to `include`:

```julia
include("path/to/my/code/wing.jl")
```

**Reply by Turbulence1116** [reply]
Yes indeed, thanks!
